UKHomeOffice/sgmr-service
# Add validation for save a vessel

To test changes

- go to _vessels_
- go to _save a vessel_
- submit form with no changes
> you should see a page level error saying "There is a problem" and nothing else in the error box
> you should see errors for Vessel name, type, usual moorings, registration only
- fill out the form and submit
- you should be returned to _vessels_ and see your new vessel

----

Testing 400 error

- follow the above, but enter a registration number you have used for another vessel
- submit form
> you should be taken to the top of the page and see the page error box with ""
